README.md

```diff
@@ -1,13 +1,14 @@
-# Extension Port Stream
+# `extension-port-stream`
 
 A module for creating a Node-style stream over a WebExtension [Runtime.Port](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/Port) object.
 
-## Breaking changes from v4 to v5
+## Installation
```

Comment:
These installation instructions are mandated by the new Yarn constraints introduced in this PR. However I've also made changes that align this README more closely with the one in the module template, adding a Contributing section.